core/src/main/java/org/apache/accumulo/core/conf/ConfigSanityCheck.java

```diff
@@ -158,8 +159,8 @@ private static void verifyValidClassName(String confOption, String className,
       Class<?> requiredBaseClass) {
     try {
       ConfigurationTypeHelper.getClassInstance(null, className, requiredBaseClass);
-    } catch (ClassNotFoundException | InstantiationException | IllegalAccessException
-        | IOException e) {
+    } catch (ClassNotFoundException | InstantiationException | IllegalAccessException | IOException
```

Comment:
By coincidence, I was also working on this today. What I ended up doing was replacing a lot of these exceptions with their parent, `ReflectiveOperationException` to remove the redundancy, and to cover the additional ones thrown by the `getDeclaredConstructor().getInstance()` pattern.

Reply:
@ctubbsii I can update to use ReflectiveOperationException. Given that you were doing similar work today would  you prefer I close this PR and wait for your updates or proceed with the changes?

Reply:
No, go ahead. My changes are larger, and merging yours in first will make mine more more manageable.

Reply:
Sounds good! Thanks.

Reply:
Merge changes. Closes #1191